### DISCUSSION TITLE
Daianna Gonzalez-Padilla - MAGMA: How does the Gene and the Gene-Set analysis operate?

### DISCUSSION THREAD

**Discussion by giscus** [lahuuki/lahuuki.github.io#2] [General]
# Daianna Gonzalez-Padilla - MAGMA: How does the Gene and the Gene-Set analysis operate?

Global overview of MAGMA Gene and Gene-Set analyses and detailed practical demonstration of the SNP-wise gene analysis and the competitive gene-set analysis.

https://daianna21.github.io/daianna_blog/posts/2024-07-16-MAGMA/

**Comment by mp8** [comment]
Perfect review and reference. Always wanted to quick reference like this one. Thanks for compiling this.

**Comment by BenxiaHu** [comment]
very good.
can you tell me how to get weight for SNP?
